app/src/main/java/io/spaceapi/community/myhackerspace/Widget.java

```diff
@@ -66,7 +69,7 @@ public void onDeleted(Context ctxt, int[] appWidgetIds) {
         for (int widgetId : appWidgetIds) {
             // Remove widget alarm
             PendingIntent pi = PendingIntent.getService(ctxt, widgetId,
-                    getIntent(ctxt, widgetId), 0);
+                        getIntent(ctxt, widgetId), SDK_INT >= Build.VERSION_CODES.S ? PendingIntent.FLAG_MUTABLE : 0);
```

Comment:
Cab you refactor the duplicated code into a method so we make sure future changes are also applied to all places?

Reply:
Yep. Done

Reply:
Nice!